content/en/testing-guide/flow_testing_may_2021.adoc

```diff
@@ -0,0 +1,94 @@
+
+== Testing Configurations
+
+A PAPR was set up with a spirometer, a battery pack, and a stopwatch.  The spirometer was attached to the system, either by a direct female to female connection, or at the end of a CPAP hose.  The system was turned on and the spirometer detached after a minute of running to determine the amount of liters of airflow during that minute.  Various different PAPR fan units, battery units, and hose configurations were tested once each.
+
+A direct spirometer configuration:
+
+[link=https://photos.smugmug.com/Tetra-Testing/2021-May-8-Papra-Build-Party/i-TWZS2dG/0/0724c2df/5K/_DSC0405-5K.jpg]
+image::https://photos.smugmug.com/Tetra-Testing/2021-May-8-Papra-Build-Party/i-TWZS2dG/0/0724c2df/L/_DSC0405-L.jpg[]
+
+
+A 3 foot bent configuration:
+
+[link=https://photos.smugmug.com/Tetra-Testing/2021-May-8-Papra-Build-Party/i-m6hHLVZ/0/33d06509/XL/_DSC0393-XL.jpg]
+image::https://photos.smugmug.com/Tetra-Testing/2021-May-8-Papra-Build-Party/i-m6hHLVZ/0/33d06509/L/_DSC0393-L.jpg[]
+
+A 6 foot direct configuration:
+
+[link=https://photos.smugmug.com/Tetra-Testing/2021-May-8-Papra-Build-Party/i-89XLt4P/0/8548c502/5K/_DSC0399-5K.jpg]
+image::https://photos.smugmug.com/Tetra-Testing/2021-May-8-Papra-Build-Party/i-89XLt4P/0/8548c502/L/_DSC0399-L.jpg[]
+
+A 6 foot bent configuration:
+
+[link=https://photos.smugmug.com/Tetra-Testing/2021-May-8-Papra-Build-Party/i-jhdVc8N/0/a086c716/5K/_DSC0401-5K.jpg]
+image::https://photos.smugmug.com/Tetra-Testing/2021-May-8-Papra-Build-Party/i-jhdVc8N/0/a086c716/L/_DSC0401-L.jpg[]
+
+A 6 foot coil configuration (unlikely to be used in the real world, considered the worst flow case):
+
+[link=https://photos.smugmug.com/Tetra-Testing/2021-May-8-Papra-Build-Party/i-39tb8Xj/0/a85ef274/L/_DSC0402-L.jpg]
+image::https://photos.smugmug.com/Tetra-Testing/2021-May-8-Papra-Build-Party/i-39tb8Xj/0/a85ef274/L/_DSC0402-L.jpg[]
+
+
+Two PAPR units were flow tested in the direct configuration, and one unit was tested with all the hosing configuration.  This second unit was tested in a direct configuration before and after hose configurations to ensure that a small reduction in battery life did not significantly impact performance, and to demonstrate that direct measurements were repeatable.  Three m12 controller units were tested with one PAPR to demonstrate that there was no change in flow attributable to the board.
+
+The fan units were:
+
+1.  OW (Orange top, white funnel, no weather shield)
+2.  GB (Grey top, black funnel)
+
+The battery units were:
+1. OG (Orange casing, grey potentiometer)
+2. GG (Grey casing, grey potentiometer)
+3. GB (Grey casing, black potentiometer)
+
+The potentiometers were set to L (low, just turned on) and H (high, turned to the maximum flow).
```

Comment:
Is there a P/N call out to this potentiometer somewhere? If so, we should probably reference it

Reply:
consider rewording _potentiometer_ to _flow control knob_ or something descriptive to the user